cmd/argo/commands/lint.go

```diff
@@ -56,7 +57,9 @@ func NewLintCommand() *cobra.Command {
 					errors.CheckError(err)
 				} else {
 					err := lint(file)
-					errors.CheckError(err)
+					if err != nil {
+						log.Warn(err)
+					}
```

Comment:
This code is incorrect. The error message for all the files would be displayed, but execution would continue. We need to both log all errors and stop execution.

Reply:
Do you mean to stop the execution on the workflow which has errors? I thought the goal with the issue where not to stop the execution?

Reply:
Oh sorry, used a misnomer. I meant that even if there are errors "Workflow manifests validated" would still be printed and exit code would still be `0`. Some of our users might have scripts that consider that a successful lint, which it wasn't.

Reply:
Cool! Then I get it, will update.